app/models/rbac.rb

```diff
@@ -96,6 +96,7 @@ def self.apply_rbac_to_associated_class?(klass)
   end
 
   def self.rbac_class(klass)
+    klass = klass.respond_to?(:klass) ? klass.klass : klass
```

Comment:
What is this line doing? Can't find where `klass.respond_to?(:klass)` would be true.